examples/config-api/config_api.out

```diff
@@ -1,43 +1,40 @@
-# Sensitive data such as passwords can be encrypted and accessed securely in code. To encrypt a value, the
-# `ballerina encrypt` command is used. It prompts the user to enter the value and a secret. In this example,
-# `ballerina` is the value and `12345` is the secret.
-ballerina encrypt
-Enter value:
-
-Enter secret:
+# This example reads two properties (i.e., `username` and `password`) from a configuration file and prints the username in the output. 
 
-Re-enter secret to verify:
+# Sensitive data you have in configuration files such as passwords should be encrypted and accessed securely in code. 
+# Therefore, first, execute the command below to encrypt the `password` property.
+ballerina encrypt
 
-Add the following to the configuration file:
-<key>="@encrypted:{njjb2H9u7Bax2FZX/X+UPZ50lR6+ckKmkesE87Yuw/x9wE1yvhcww1ehOc+mjgCc}"}"
+# This prompts to enter the value (i.e., the password) and a secret (i.e., the key).
```

Comment:
The secret is *not* the key! Those two are completely separate and unrelated.

Reply:
Fixed.